config/lambdatest/wdio.shared.lambdatest.conf.ts

```diff
@@ -0,0 +1,25 @@
+import { config as baseConfig } from '../wdio.shared.conf.js';
+
+// Shared configuration for LambdaTest across all environments
+export const config: WebdriverIO.Config = {
+    ...baseConfig,
+
+    // LambdaTest user credentials
+    user: process.env.LT_USERNAME || "YOUR_USERNAME",
+    key: process.env.LT_ACCESS_KEY || "YOUR_ACCESS_KEY",
+    path: "/wd/hub",
+
+    // Default settings for all environments
```

Comment:
These options are already in the shared configuration, so they can be removed from the LT shared configuration